src/common/constants.ts

```diff
@@ -8,3 +10,10 @@ export const FILES_PATHS = {
   OUTLINED_TEMP: 'svg-outlined-temp/',
   FILLED_TEMP: 'svg-temp/'
 }
+
+// *********************************
+// Directories used in APIs (V1)
+// *********************************
+
+export const tempDirectory = path.join(__dirname, '..', '/temp/')
```

Comment:
lets remove the 2nd slash and keep them like `/temp`. The second slash should come from sub directories, just like routes.